lib/rules/no-useless-computed-key.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
 
         return {
             Property: check,
+            PropertyDefinition: enforceForClassMembers ? check : lodash.noop,
```

Comment:
We should add something like `|| node.type === "PropertyDefinition"` on line 64:

```js
/* eslint no-useless-computed-key: ["error", { enforceForClassMembers: true }] */

class C {
    ["constructor"]; // false positive
    static ["prototype"]; // false positive
    ["__proto__"]; // false negative
}
```

(also, `lodash` has been removed in the meantime)

Reply:
`static ["constructor"]` is also false positive (produces parsing error instead of a runtime error in a maybe-unused code).

For some reason, "constructor" is allowed as the name of a static method, but not as the name of a static field.

> ClassElement : static MethodDefinition
> * It is a Syntax Error if PropName of MethodDefinition is "prototype".
>
> ClassElement : static FieldDefinition ;
> * It is a Syntax Error if PropName of FieldDefinition is "prototype" or "constructor".